sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactsView.java

```diff
@@ -463,11 +463,14 @@ public HorizontalLayout createStatusFilterBar() {
 				scrollLayout.setMargin(false);
 
 				DateField toReferenceDate = new DateField(I18nProperties.getCaption(Captions.to), LocalDate.now());
+				toReferenceDate.setId("toReferenceDateField");
```

Comment:
Please use ButtonHelper for creating buttons, that will assign the id to the button

Reply:
ok